### DISCUSSION TITLE
1.0.0.0.0.0.0.0.0

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#39] [Announcements]
We've launched! 🚀

New with this release:

* You can now [overload variables](https://github.com/TodePond/DreamBerd#overloading)! (thanks to @scolsen)
* You can now [overload variables](https://github.com/TodePond/DreamBerd#overloading)!!! (thanks to @scolsen)
* You can now *get low* and [work with memory](https://github.com/TodePond/DreamBerd#memory) (thanks to @scolsen)
* [Github privacy protection](https://github.com/TodePond/DreamBerd#copilot)!

And most importantly...
You can now [run DreamBerd](https://github.com/TodePond/DreamBerd#compiling)!

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/1.0.0.0.0.0.0.0.0'>1.0.0.0.0.0.0.0.0</a>.</em>

**Comment by tif-calin** [comment]
overloading variables is cool, but can we add a way to download variables?

**Reply by TodePond** [reply]
yes

**Comment by forgdeer** [comment]
can we upload variables instead?

**Reply by TodePond** [reply]
yes 🫡
